lib/corm/validations.rb

```diff
@@ -0,0 +1,56 @@
+# encoding: utf-8
+
+module Corm
+  module Validations
+
+    def there_are_too_many_keys_requested?(key_values)
+      if key_values.keys.count > self.primary_key_count
+        return "You defined more find keys than the primary keys of the table!"
```

Comment:
I just want to be clear that, right now, this error message is not used by the corresponding exception... not yet.